src/inference/hand_detection.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import cv2
+import sys
+import tensorflow as tf
+import numpy as np
+
+from yolo import YOLO
+
+
+class HandDetector:
+
+    def __init__(self):
+        self.yolo = YOLO("models/cross-hands-tiny.cfg", "models/cross-hands-tiny.weights", ["hand"])
+        self.yolo.size = int(416)
+        self.yolo.confidence = float(0.2)
+
+    def predict(self, img, draw):
+        img_resize = cv2.resize(img, (416, 416))
+
+        width, height, inference_time, results = self.yolo.inference(img_resize)
+
+        conf_sum = 0
+        detection_count = 0
+
+        boxes = []
+
+        for detection in results:
+            id, name, confidence, x, y, w, h = detection
+            boxes.append((x, y, w, h))
+
+            if draw_bb:
```

Comment:
draw_bb is not defined in this scope, so I guess this script will not even work :p